### ISSUE TITLE
[Serious] Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#80]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87916 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 Objects page |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/objects?state=A> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/17/26 at 12:14 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The contrast ratio between text and its background is not at least 4.5:1.
Select an Object from the table, on Object Detail page, in Active Files section, expand one of the rows that has a URL listed.  That URL is light blue text on off white background.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

How To Fix: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Source Code

```
<a href="https://s3.us-east-2.amazonaws.com/aptrust.test.preservation.glacier-deep.oh/e6623125-e7cf-443a-801d-fa59cb016007">https://s3.us-east-2.amazonaws.com/aptrust.test.preservation.glacier-deep.oh/e6623125-e7cf-443a-801d-fa59cb016007</a>
```

## Screenshots

![03_blue offwhite contrast](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87916/03_blue_offwhite_contrast.png)
